<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PdfMarginAdder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBoxInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>210</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditPathArchivoInput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Archivo de Salida</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButtonProcess">
           <property name="text">
            <string>Process</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditPathArchivoOutput"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonSelectFileInput">
           <property name="text">
            <string>Seleccionar
archivo
de entrada</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButtonSelectFileOutput">
           <property name="text">
            <string>Seleccionar
archivo
de salida</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelArchivo">
           <property name="text">
            <string>Archivo de Entrada</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutParameters" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Parametros</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxFolder">
             <property name="text">
              <string>Seleccionar Carpeta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonShowLog">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Mostrar Log</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxMargenes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Margenes</string>
          </property>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>125</x>
             <y>44</y>
             <width>61</width>
             <height>60</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditLeft">
           <property name="geometry">
            <rect>
             <x>88</x>
             <y>63</y>
             <width>30</width>
             <height>20</height>
            </rect>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditRight">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>63</y>
             <width>30</width>
             <height>20</height>
            </rect>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditTop">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>20</y>
             <width>30</width>
             <height>20</height>
            </rect>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditBottom">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>110</y>
             <width>30</width>
             <height>20</height>
            </rect>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacerLog">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frameLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTextEdit" name="textEditLog"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
